<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSettingsWindow</class>
 <widget class="QWidget" name="FilterSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>General flags </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="filterFlags">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QCheckBox" name="filterModesCheck">
          <property name="text">
           <string>Game mode(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="filterModes">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="serverNameCheck">
            <property name="text">
             <string>Server name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="serverNameInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="serverAddressCheck">
            <property name="text">
             <string>Server address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="serverAddressInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="serverIWADCheck">
            <property name="text">
             <string>IWAD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="serverIWADInput"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>PWADs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pwadWildcards">
          <property name="text">
           <string>Allow wildcards in PWADs</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLineEdit" name="pwadAddInput"/>
            </item>
            <item>
             <widget class="QListWidget" name="pwadListInput"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QPushButton" name="pwadAddButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pwadDelButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="dmflagsCheck">
            <property name="text">
             <string>DMFlags</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dmflagsInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="dmflags2Check">
            <property name="text">
             <string>DMFlags2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dmflags2Input"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="zadmflagsCheck">
            <property name="text">
             <string>ZADMFlags</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="zadmflagsInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="compatflagsCheck">
            <property name="text">
             <string>CompatFlags</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="compatflagsInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="zacompatflagsCheck">
            <property name="text">
             <string>ZACompatFlags</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="zacompatflagsInput"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
